packages/vats/test/upgrading/bootstrapVatUpgrade.js

```diff
@@ -0,0 +1,61 @@
+// @ts-check
+import { E, Far } from '@endo/far';
+import { makePromiseKit } from '@endo/promise-kit';
+
+const bundles = {
+  board: 'board',
+};
+
+export const buildRootObject = () => {
+  const bootKit = makePromiseKit();
+  const v1Kit = makePromiseKit();
+
+  return Far('B', {
+    bootstrap: async (vats, devices) => {
+      // console.debug('1 createVatAdminService');
+      assert(vats.vatAdmin);
+      assert(devices.vatAdmin);
+      const vatAdmin = await E(vats.vatAdmin).createVatAdminService(
+        devices.vatAdmin,
+      );
+
+      // console.debug('2 getBundleIDByName');
+      const bundleId = await E(vatAdmin).getBundleIDByName(bundles.board);
+      assert(bundleId, 'bundleId must not be empty');
+      const bundleCap = await E(vatAdmin).getBundleCap(bundleId);
+
+      // console.debug('3 return from bootstrap()');
+      bootKit.resolve({ vatAdmin, bundleId, bundleCap });
+    },
+
+    startBoardV1: async () => {
+      const { vatAdmin, bundleCap } = await bootKit.promise;
+      const vatInfo = await E(vatAdmin).createVat(bundleCap);
+
+      // console.debug('vatInfo', [...Object.keys(vatInfo)]);
+      const thing = Far('Thing', {});
+      const board = E(vatInfo.root).getBoard();
+      const boardId = await E(board).getId(thing);
+
+      // console.debug('before upgrade', { boardId });
+      v1Kit.resolve({ vatInfo, thing, boardId });
+      return true;
+    },
+
+    nullBoardUpgrade: async () => {
```

Comment:
nit: "null upgrade" seems like the verb and "board" the direct object.
consider, `nullUpgradeBoardV1`